# AutoTest results for test_store (12/12/2012 11:42)
TEST_DATA_CORRECTNESS.test_quotes_in_string: pass
TEST_DATA_CORRECTNESS.test_trailing_blanks: pass
TEST_ERROR_MESSAGE.test_error_message: pass
TEST_EXTENDED_SELECT.test_extended_select: pass
TEST_GUID_TYPE.test_uuid_type: FAIL (Number of results is not expected)
	on_prepare: ok
	test routine: exceptional (Developer exception in EXCEPTIONS.raise)
		******************************** Thread exception *****************************
		In thread           Root thread            0x0 (thread id)
		*******************************************************************************
		-------------------------------------------------------------------------------
		Class / Object      Routine                Nature of exception           Effect
		-------------------------------------------------------------------------------
		ISE_EXCEPTION_MANAGER
		                    developer_raise        Number of results is not expected:
		<00000000025DD0D8>                         Developer exception.          Fail
		-------------------------------------------------------------------------------
		ISE_EXCEPTION_MANAGER
		                    developer_raise        
		<00000000025DD0D8>                         Routine failure.              Fail
		-------------------------------------------------------------------------------
		ISE_EXCEPTION_MANAGER
		                    raise @11              
		<00000000025DD0D8>                         Routine failure.              Fail
		-------------------------------------------------------------------------------
		DEVELOPER_EXCEPTION raise @2                                            
		<00000000025E4548>  (From EXCEPTION)       Routine failure.              Fail
		-------------------------------------------------------------------------------
		EQA_ASSERTIONS      raise @3                                            
		<00000000025DFFF8>  (From EXCEPTIONS)      Routine failure.              Fail
		-------------------------------------------------------------------------------
		EQA_ASSERTIONS      assert @5                                           
		<00000000025DFFF8>                         Routine failure.              Fail
		-------------------------------------------------------------------------------
		TEST_GUID_TYPE      assert @2                                           
		<00000000025DFF58>  (From EQA_TEST_SET)    Routine failure.              Fail
		-------------------------------------------------------------------------------
		TEST_GUID_TYPE      uuid_type_make_selection @6
		<00000000025DFF58>                         Routine failure.              Fail
		-------------------------------------------------------------------------------
		TEST_GUID_TYPE      test_uuid_type @6                                   
		<00000000025DFF58>                         Routine failure.              Fail
		-------------------------------------------------------------------------------
	on_clean: ok
TEST_LENGTH_OF_COLUMN_NAME.test: pass
TEST_MULTIPLE_THREAD.test_multiple_thread: unresolved (Evaluator Died)
	The evaluator process which was executing test TEST_MULTIPLE_THREAD.test_multiple_thread terminated unexpectedly with exit code 1. Any output from the
	evaluator is printed below.
	
	

	test_store: system execution failed.

	Following is the set of recorded exceptions:

	

	******************************** Thread exception *****************************

	In thread           Child thread           0x1E4 (thread id)

	*******************************************************************************

	-------------------------------------------------------------------------------

	Class / Object      Routine                Nature of exception           Effect

	-------------------------------------------------------------------------------

	DATA_OPERATION_THREAD

	                    execute_action @5      result_expected:

	<000000000277F548>                         Assertion violated.           Fail

	-------------------------------------------------------------------------------

	DATA_OPERATION_THREAD

	                    execute_action @5      

	<000000000277F548>                         Routine failure.              Fail

	-------------------------------------------------------------------------------

	DB_SELECTION        load_result @27                                     

	<0000000002886FB8>                         Routine failure.              Fail

	-------------------------------------------------------------------------------

	DATA_OPERATION_THREAD

	                    make_selection @3      

	<000000000277F548>                         Routine failure.              Fail

	-------------------------------------------------------------------------------

	DATA_OPERATION_THREAD

	                    execute @16            

	<000000000277F548>                         Routine failure.              Fail

	-------------------------------------------------------------------------------

	DATA_OPERATION_THREAD

	                    thr_main @3            

	<000000000277F548>  (From THREAD)          Routine failure.              Rescue

	-------------------------------------------------------------------------------

	EQA_EVALUATOR       root's creation                                     

	<0000000002750588>                         Routine failure.              Exit

	-------------------------------------------------------------------------------

TEST_NESTED_TRANSACTION.test: pass
TEST_NUMERIC_TYPES.test_numeric_types: pass
TEST_NUMERIC_TYPES.test_numeric_types_use_decimal_mode: FAIL (Result is not expected)
	on_prepare: ok
	test routine: exceptional (Developer exception in EXCEPTIONS.raise)
		******************************** Thread exception *****************************
		In thread           Root thread            0x0 (thread id)
		*******************************************************************************
		-------------------------------------------------------------------------------
		Class / Object      Routine                Nature of exception           Effect
		-------------------------------------------------------------------------------
		ISE_EXCEPTION_MANAGER
		                    developer_raise        Result is not expected:
		<00000000025DD0D8>                         Developer exception.          Fail
		-------------------------------------------------------------------------------
		ISE_EXCEPTION_MANAGER
		                    developer_raise        
		<00000000025DD0D8>                         Routine failure.              Fail
		-------------------------------------------------------------------------------
		ISE_EXCEPTION_MANAGER
		                    raise @11              
		<00000000025DD0D8>                         Routine failure.              Fail
		-------------------------------------------------------------------------------
		DEVELOPER_EXCEPTION raise @2                                            
		<00000000025DD048>  (From EXCEPTION)       Routine failure.              Fail
		-------------------------------------------------------------------------------
		EQA_ASSERTIONS      raise @3                                            
		<00000000025B3BE8>  (From EXCEPTIONS)      Routine failure.              Fail
		-------------------------------------------------------------------------------
		EQA_ASSERTIONS      assert @5                                           
		<00000000025B3BE8>                         Routine failure.              Fail
		-------------------------------------------------------------------------------
		TEST_NUMERIC_TYPES  assert @2                                           
		<00000000025B3B48>  (From EQA_TEST_SET)    Routine failure.              Fail
		-------------------------------------------------------------------------------
		TEST_NUMERIC_TYPES  numeric_types_with_decimal_make_selection @17
		<00000000025B3B48>                         Routine failure.              Fail
		-------------------------------------------------------------------------------
		TEST_NUMERIC_TYPES  test_numeric_types_use_decimal_mode @8
		<00000000025B3B48>                         Routine failure.              Fail
		-------------------------------------------------------------------------------
	on_clean: ok
TEST_PROCEDURE.test_procedure: pass
TEST_SELECT.test_basic_select: pass
TEST_SELECT.test_large_string_select: pass
TEST_TRANSACTION.test: pass
